### ISSUE TITLE
Flag preset names are unlocalized

### ISSUE THREAD

**Issue by 1ec5** [osmlab/name-suggestion-index#5982]
The flags tree has `displayName`s in English only. This results in a suboptimal experience for users of languages other than English, who have to know the name of the flag in English (or the ISO country code of a national flag). Unlike a brand or operator, a localized flag name is the norm, not the exception. The way NSI is currently set up, we would basically need to multiply the number of `man_made/flagpole` entries by the number of supported languages.

Maybe the project could instead pull the names automatically from the associated `flag:wikidata` or `subject:wikidata` QID as an occasional build step? Alternatively, NSI could adopt a localization system similar to the one used by id-tagging-schema – that is, as part of the iD project. But that would be unnecessarily complicated for the other trees, and it’d be busywork for iD translators to retranslate basic things like country names that are already available elsewhere.

/ref https://github.com/osmlab/name-suggestion-index/pull/5960#discussion_r776572455

**Comment by bhousel** [comment]
I dunno, I _just_ said I don't want to introduce localization here. 
Multiplying the current 384 flags * the current 90ish languages = translation of 34k+ strings, and we're nowhere close to tracking all the flags that people might want to map.

**Comment by 1ec5** [comment]
Right, I agree with you that introducing localization here isn’t great. So how do we solve this problem, either here or in some other project? Should flags move to id-tagging-schema?

**Comment by bhousel** [comment]
I dont know, there might not be a solution for this. I agree that leveraging the existing translations in Wikidata is the way to go. 

Also I dont really have any authority over those other projects anymore, so I think it's time to move on and not think about solving problems there..  I need some time to think about it.  (Put another way, it seems pretty clear to me that dumping a load of flags into `id-tagging-schema` and asking people to translate them all is not the answer).
